pkg/spdk/engine.go

```diff
@@ -353,7 +353,13 @@ func (e *Engine) ValidateAndUpdate(
 		if err != nil {
 			return err
 		}
+
 		if err := initiator.LoadNVMeDeviceInfo(); err != nil {
+			if strings.Contains(err.Error(), "connecting state") ||
+				strings.Contains(err.Error(), "resetting state") {
+				e.log.WithError(err).Warnf("Failed to validate and update engine %s", e.Name)
```

Comment:
`Ignored to valide and update engine..., because the device is still in a transient state.`

Reply:
Updated, thanks.